application/comit_node/src/http_api/rfc003/swap.rs

```diff
@@ -61,6 +64,14 @@ pub struct SwapRequestBody<AL: Ledger, BL: Ledger, AA: Asset, BA: Asset> {
     alpha_ledger_lock_duration: AL::LockDuration,
     #[serde(flatten)]
     identities: SwapRequestBodyIdentities<AL::Identity, BL::Identity>,
+    #[serde(rename = "counterparty_ip_address")]
+    bob_ip_address: IpAddr,
+    #[serde(default = "default_port", rename = "counterparty_port")]
+    bob_port: u16,
+}
+
+fn default_port() -> u16 {
+    8001
```

Comment:
❌ this default port is a big decision. We definitely need to come up with something more original and commonly used than 8001.
Idea hats on!

Reply:
Good idea, but the UTF-8 code for :chains: appears to be [`26d3`](http://www.unicode.org/emoji/charts/full-emoji-list.html#26d3).

Reply:
Yes, but ports are decimals hence `9939`